### ISSUE TITLE
Accessibility: "How to build a bot" section of Welcome Page [Regression]

### ISSUE THREAD

**Issue by corinagum** [Microsoft/BotFramework-Emulator#1096]
## Version
Latest master

## Describe the bug
"How to build a bot" section on the welcome page is no longer keyboard accessible. This is a regression from previous functionality

## To Reproduce
1. Open Emulator
2. Tab through the different buttons and links on the Welcome Page
3. Note that after "Sign in with your Azure account", the keyboard focus jumps back to the Bot Explorer button instead of going to "How to built a bot" section

## Expected behavior
After tabbing to "Sign in with your azure account", the next keyboard focus item should be "design guidelines".

[bug]

**Comment by corinagum** [comment]
o	[Jen] This is a P0/AccSelf Red.  Would like more details about if the regression happened with the whole page or just a few items on the page.
o	[Jason]  These links are not actually links.  They are showing up semantically as clickable text.  
How found:           Mouse move (2654,337)
hwnd=0x00000000000D0E5C 32bit class="Chrome_RenderWidgetHostHWND" style=0x56300000 ex=0x20
Name:                                        "More about working locally with a bot."
ControlType:                             UIA_TextControlTypeId (0xC364)
LocalizedControlType:           "text"
